config/scheduled_tasks.yml

```diff
@@ -0,0 +1,53 @@
+UpdateCgi:
+  cron: '0 0 1 * *'
```

Comment:
One thing that might be worth doing here that I tried to do with the civic scheduler is space the jobs out some so that there's not like one specific period where several of the jobs are all running at the same time and making the box slow. So maybe like if they're weekly some go on Saturday and some go on Tuesday or whatever.